ankh/cli.py

```diff
@@ -24,36 +26,93 @@
 import logging
 
 from ankh.utils import command_header
-from ankh.utils import stage_print
 from ankh.kubectl import kubectl_action
 
 logger = logging.getLogger('ankh')
+valid_commands = [ 'apply', 'template', 'config', 'current-context', 'set-context' ]
+default_config_files = [ 'ankh.yaml', 'deploy.yaml' ]
+
+def ankh_config_read(args, log=False):
+    # Kube context may be present in this user's ankh config. Check for it.
+    if os.path.exists(args.ankhconfig):
+        with open(args.ankhconfig, 'r') as f:
+            config = yaml.safe_load(f)
+            if 'current-context' in config:
```

Comment:
Shouldn't we just expect `current_context` to be in the config yaml directly?

Reply:
are you saying we should expect current-context as a top-level key in ~/.ankh/config (or $ANKHCONFIG / --ankhconfig)? that should be the case, but we're also resilient to that key being missing. you can add that key manually or use set-context which will write the key in for you. is that what you meant?

Reply:
Sorry, I wasn't very clear on my comment. I just meant it should have an underscore instead of a dash

Reply:
kubectl's ~/.kube/config uses 'current-context' so I went with that